androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt

```diff
@@ -252,6 +255,15 @@ class ComposePrototypeUITest : BaseTests() {
         }
     }
 
+    fun clearConversation() {
+        messenger {
+            verifyPageIsVisible()
+            enterCommand(clearConversation)
+            waitForProperResponse(clearConversationMessage)
```

Comment:
can we check that when `client.clearConversation` is called Event.ConversationCleared is dispatched by Transport?